contrib/apelscripts/condor_batch_blah.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+fail () {
+    echo "$@" >&2
+    exit 1
+}
+
+safe_config_val () {
+    var=$1
+    attr=$2
+    val=$(condor_ce_config_val $attr) ||
+    fail "Failed to retrieve CE configuration value '$attr'"
+    eval "$var"='$val'
+}
+
+OUTPUT_DIR="$(condor_ce_config_val APEL_OUTPUT_DIR)"
+[[ -d $OUTPUT_DIR && -w $OUTPUT_DIR ]] || fail "Cannot write to $OUTPUT_DIR"
+OUTPUT_DATETIME=`date '+%Y%m%d-%H%M'`
+
+HISTORY_DIR=`condor_config_val PER_JOB_HISTORY_DIR`
```

Comment:
I forget exactly why @edquist had to implement `safe_config_val` but we probably should do the same thing for `condor_config_val`

Reply:
> I forget exactly why @edquist had to implement `safe_config_val` but we probably should do the same thing for `condor_config_val`

Heard my name!

Looks like the idea behind `safe_config_val` was to provide a single function to get a config val if it exists, otherwise cause the script fail.

@brianhlin, it looks like you introduced `safe_config_val` originally [here](https://github.com/htcondor/htcondor-ce/commit/d8277b21508c96ec09782fbb0204b68b63e9dee0#diff-de4e3077e8b9dd3874333d5c4124d541d39f719fd42a5a99bd02e8567aec0c0eR3-R5)

but i modified it later (in 8a8c052992a163306bd204b410a2e6c7212c6fa7) because the `exit 1` was happening inside a subshell and not getting propagated to the main script process

Reply:
@brianhlin So I'm assuming I should make a second function called `safe_ce_config_val()` and change `safe_config_val()` to call `condor_config_val`?

Reply:
Yeah, that sounds reasonable